platform/tests/Platform.ApiTests/Steps/EstablishmentTrustsSteps.cs

```diff
@@ -35,118 +32,139 @@ private void GivenAValidTrustRequestWithId(string id)
+    private void GivenAnInvalidValidTrustRequestWithId(string id)
     {
-        await _api.Send();
+        _api.CreateRequest(RequestKey, new HttpRequestMessage
+        {
+            RequestUri = new Uri($"/api/trust/{id}", UriKind.Relative),
+            Method = HttpMethod.Get
+        });
     }
 
-    [Then("the trust result should be ok")]
-    private void ThenTheTrustResultShouldBeOk()
+    [Given("a valid trust suggest request with searchText '(.*)")]
+    private void GivenAValidTrustsSuggestRequest(string searchText)
     {
-        var result = _api[RequestKey].Response;
+        var content = new { SearchText = searchText, Size = 5, SuggesterName = "trust-suggester" };
 
-        result.Should().NotBeNull();
-        result.StatusCode.Should().Be(HttpStatusCode.OK);
+        _api.CreateRequest(SuggestRequestKey, new HttpRequestMessage
+        {
+            RequestUri = new Uri("/api/trusts/suggest", UriKind.Relative),
+            Method = HttpMethod.Post,
+            Content = new StringContent(content.ToJson(), Encoding.UTF8, "application/json")
+        });
     }
 
-    [Given("a valid trusts query request")]
-    private void GivenAValidTrustsQueryRequest()
+    [Given("an invalid trust suggest request")]
+    private void GivenAnInvalidTrustsSuggestRequest()
     {
-        _api.CreateRequest(QueryRequestKey, new HttpRequestMessage
+        var content = new { Size = 0 };
+
+        _api.CreateRequest(SuggestRequestKey, new HttpRequestMessage
         {
-            RequestUri = new Uri("/api/trusts", UriKind.Relative),
-            Method = HttpMethod.Get
+            RequestUri = new Uri("/api/trusts/suggest", UriKind.Relative),
+            Method = HttpMethod.Post,
+            Content = new StringContent(content.ToJson(), Encoding.UTF8, "application/json")
         });
     }
 
-    [Then("the trusts query result should be ok")]
-    private void ThenTheTrustsQueryResultShouldBeOk()
+    [When("I submit the trust request")]
+    private async Task WhenISubmitTheTrustsRequest()
     {
-        var result = _api[QueryRequestKey].Response;
-
-        result.Should().NotBeNull();
-        result.StatusCode.Should().Be(HttpStatusCode.OK);
+        await _api.Send();
     }
 
-    [Given("a valid trusts search request")]
-    private void GivenAValidTrustsSearchRequest()
+    [Then("the trust result should be correct")]
+    private async Task ThenTheTrustResultShouldBeCorrect()
     {
-        _api.CreateRequest(SearchRequestKey, new HttpRequestMessage
-        {
-            RequestUri = new Uri("/api/trusts/search", UriKind.Relative),
-            Method = HttpMethod.Post
-        });
+        var response = _api[RequestKey].Response;
+
+        response.Should().NotBeNull();
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        var content = await response.Content.ReadAsByteArrayAsync();
+        var result = content.FromJson<Trust>();
+
+        result.CompanyNumber.Should().Be("7539918");
```

Comment:
Please can the asserted values move to the feature using `table.CompareToInstance()`?